core/src/main/java/com/sequenceiq/cloudbreak/core/flow/service/ReactorFlowManager.java

```diff
@@ -187,7 +188,9 @@ public void triggerClusterUpscale(Object object) {
     public void triggerClusterDownscale(Object object) {
         UpdateAmbariHostsRequest request = (UpdateAmbariHostsRequest) object;
         ClusterScalingContext context = new ClusterScalingContext(request);
-        reactor.notify(FlowPhases.CLUSTER_DOWNSCALE.name(), eventFactory.createEvent(context, FlowPhases.CLUSTER_DOWNSCALE.name()));
+        FlowPhases phase =
+                ScalingType.DOWNSCALE_ONLY_CLUSTER == request.getScalingType() ? FlowPhases.CLUSTER_DOWNSCALE : FlowPhases.CLUSTER_AND_STACK_DOWNSCALE;
+        reactor.notify(phase.name(), eventFactory.createEvent(context, FlowPhases.CLUSTER_DOWNSCALE.name()));
```

Comment:
First phase.name() then FlowPhases.CLUSTER_DOWNSCALE in the same expression. What's the difference in this case?